tests/test_tags.py

```diff
@@ -1226,3 +1226,15 @@ def test_pypy_first_none_any_tag(self, monkeypatch):
                 break
 
         assert tag == tags.Tag("pp3", "none", "any")
+
+    def test_cpython_first_none_any_tag(self, monkeypatch):
+        # When building the complete list of cpython tags, make sure the first
+        # <interpreter>-none-any one is cpxx-none-any
+        monkeypatch.setattr(tags, "interpreter_name", lambda: "cp")
+
+        for tag in tags.sys_tags():
+            if tag.abi == "none" and tag.platform == "any":
+                break
+
+        interpreter = "cp" + tags._version_nodot(sys.version_info[:2])
```

Comment:
There's no need to avoid APIs that are tested elsewhere and thus considered functioning.

```suggestion
        interpreter = f"cp{tags.interpreter_version()}"
```